### PR TITLE
doc: add sample for pushing non JSON docs

### PR DESCRIPTION
**Pull request by louis-bompart**

### CODE REVIEW

**Review by y-lakhdar — CHANGES_REQUESTED** [review]
-  [ ] Manual tests

File: samples/pushBatchOfNonJSONDocuments.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    new DocumentBuilder(`${baseRemoteUrl}/${fileName}`, fileName)
+      .withCompressedBinaryData(
+        prepareDocument(join(baseLocalDirectory, fileName)),
+        'GZIP'
+      )
+      .withFileExtension(parse(fileName).ext)
+  );
+
+  const myBatchOfDocuments = {
+    addOrUpdate: myDocuments,
```
[the remark this answers is not in the document]
Comment (louis-bompart) [reply samples/pushBatchOfNonJSONDocuments.ts:23]: > The delete array is mandatory

Well. It worked without on my machine

[the remark this answers is not in the document]
Comment (y-lakhdar) [reply samples/pushBatchOfNonJSONDocuments.ts:23]: without getting typing errors?

[the remark this answers is not in the document]
Comment (louis-bompart) [reply samples/pushBatchOfNonJSONDocuments.ts:27]: Oupps nvm, fixed.

**Review by y-lakhdar — APPROVED** [review]

**Pull request merged**